consensus/bundle_validator/bundle_validator.c

```diff
@@ -38,15 +38,18 @@ retcode_t bundle_validate(const tangle_t* const tangle, trit_array_p tail_hash,
   retcode_t res = RC_OK;
   iota_transaction_t curr_tx;
 
+  *is_valid = true;
```

Comment:
Isn't it easier/safer to initialise it to `false` ? Then you don't have to put `*is_valid = false;` on every error ?

Reply:
Whoops, sorry for resolving - mobile.
We.never set the value to true?

Reply:
yes, that's actually better

Reply:
In my head `res = validate_signature(bundle, normalized_bundle_trits, is_valid);` would have set the value to true, but it doesn't actually

Reply:
Yes, I just saw that as well. Will fix too.
As for setting is_valid to false, it's more difficult to set it to true later, because of the loop. Let's say that I like it's more readable IMMO.